### PR TITLE
added support of XGBClassifier

### PR DESCRIPTION
**Pull request by nitesh585**
#922

### CODE REVIEW

File: model/xgboost/dffml_model_xgboost/xgbclassifier.py [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+            n_jobs=self.config.n_jobs,
+            colsample_bytree=self.config.colsample_bytree,
+            booster=self.config.booster,
+            min_child_weight=self.config.min_child_weight,
+            reg_lambda=self.config.reg_lambda,
+            reg_alpha=self.config.reg_alpha,
+        )
+
+        self.saved.fit(x_data, y_data, eval_metric="merror")
+        print("\nTraining Successfully !!")
```
Comment (pdxjohnny) [inline model/xgboost/dffml_model_xgboost/xgbclassifier.py:212]:
```suggestion
```

**Comment by pdxjohnny** [comment]
The squashed commit message for this PR would be: 


```
model: xgboost: Added XGBClassifier

Fixes: #922 
```

**Comment by pdxjohnny** [comment]
Please rebase master into this then squash and I'll merge

**Comment by nitesh585** [comment]
> Please rebase master into this then squash and I'll merge

@pdxjohnny I did all those steps and now, It's ready to merge.

**Pull request merged**